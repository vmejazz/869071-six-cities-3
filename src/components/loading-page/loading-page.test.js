import React from "react";
import renderer from "react-test-renderer";
import LoadingPage from "./loading-page.jsx";

it(`<LoadingPage /> should render LoadingPage page `, () => {
  const tree = renderer
      .create(
          <LoadingPage
          />
      )
      .toJSON();

  expect(tree).toMatchSnapshot();
});
