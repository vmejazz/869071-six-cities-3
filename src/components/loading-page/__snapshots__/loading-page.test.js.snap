// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingPage /> should render LoadingPage page  1`] = `
Array [
  <h1
    style={
      Object {
        "fontSize": "28px",
        "marginTop": "80px",
        "textAlign": "center",
      }
    }
  >
    Приложение загружается ...
  </h1>,
  <div>
    <svg
      display="block"
      height="200"
      preserveAspectRatio="xMidYMid"
      style={
        Object {
          "margin": "auto",
        }
      }
      viewBox="0 0 100 100"
      width="200"
      xmlns="http://www.w3.org/2000/svg"
    >
      <circle
        cx="50"
        cy="50"
        fill="none"
        r="35"
        stroke="#99adba"
        strokeDasharray="164.93361431346415 56.97787143782138"
        strokeWidth="10"
        transform="rotate(99.747 50 50)"
      >
        <animateTransform
          attributeName="transform"
          dur="1s"
          keyTimes="0;1"
          repeatCount="indefinite"
          type="rotate"
          values="0 50 50;360 50 50"
        />
      </circle>
    </svg>
  </div>,
]
`;
